tests/spec/smoketest-eq-services-spec.js

```diff
@@ -0,0 +1,68 @@
+const AuthorHome = require('../pages/AuthorHome.page');
+const CreateQuestionnairePage = require('../pages/CreateQuestionnaire.page');
+const DesignQuestionnairePage = require('../pages/DesignQuestionnaire.page');
+
+const BASE_URL = process.env.BASE_URL || 'http://localhost:3000';
+
+describe('eQ Services Smoke Test', () => {
+  beforeAll(() => {
+    browser.url(BASE_URL);
+    browser.click(AuthorHome.createQuestionnaireButton());
+    browser.setValue(CreateQuestionnairePage.questionnaireTitle(), 'title');
+    browser.setValue(CreateQuestionnairePage.questionnaireDescription(), 'description');
+    browser.selectByVisibleText(CreateQuestionnairePage.questionnaireTheme(), 'default');
+    browser.selectByVisibleText(CreateQuestionnairePage.questionnaireLegalBasis(), 'StatisticsOfTradeAct');
+    browser.click(CreateQuestionnairePage.navigationToggle());
+    browser.click(CreateQuestionnairePage.createButton());
+  });
+
+  it('should update navigation title when section title changed', () => {
+    browser.waitForExist("h3[class*='SectionTitle']");
+    expect(browser.getText(DesignQuestionnairePage.getFirstSectionTitle())).toEqual('Section Title');
```

Comment:
You shouldn't ever be needing to start a scenario with an assert. This just highlights that the before does too much